modules/installation-osp-api-scaling.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * networking/openstack/load-balancing-openstack.adoc
+
+[id="installation-osp-api-scaling_{context}"]
+= Scaling clusters by using Octavia
+
+If you want to use multiple API load balancers, or if your cluster does not use Kuryr, create an Octavia load balancer and then configure your cluster to use it.
+
+.Prerequisites
+
+* Octavia is available on your {rh-openstack} deployment.
```

Comment:
s/{rh-openstack}/{rh-openstack-first}

{rh-openstack-first} was used in the first module, but I think we're supposed to treat every module as a stand-alone instance. (If I'm wrong, please feel free to ignore this and the following suggestions.)